tests/lib/jquery.ui/jquery.ui.inputextender.tests.js

```diff
@@ -122,10 +119,14 @@
 		var $input = $( '<input/>' ).appendTo( $( 'body' ) ).focus();
 		var extender = newTestInputextender( $input );
 
-		assert.ok(
-			extender.extensionIsActive(),
-			'Extension active initially because input has focus. ' + BROWSER_FOCUS_NOTE
-		);
+		if( !extender.extensionIsActive() && 'hidden' in document && document.hidden ) {
```

Comment:
Can you please introduce a variable instead of calling `extender.extensionIsActive()` twice?